### ISSUE TITLE
No-Root Method ?

### ISSUE THREAD

**Issue by Hawk2811** [diejuse/chroot_Maemo-leste_on_Android#2]
Would it be possible to do this without root?

**Comment by diejuse** [comment]
Yes, it's possible. But I haven't tried it, and it probably needs some extra steps. Give it a try.

**Comment by Hawk2811** [comment]
> Yes, it's possible. But I haven't tried it, and it probably needs some extra steps. Give it a try.

I made several attempts using proot but it didn't work I can enter the system shell but I can't start the interface

**Comment by diejuse** [comment]
Sonn will be possible.
